tests/cmdline/commands/test_storage.py

```diff
@@ -23,6 +23,26 @@ def tests_storage_version(run_cli_command):
     assert version in result.output
 
 
+@pytest.mark.parametrize(
+    'exception_cls, exit_code',
+    (
+        (exceptions.CorruptStorage, 3),
+        (exceptions.UnreachableStorage, 3),
+        (exceptions.IncompatibleStorageSchema, 4),
+    ),
+)
+def tests_storage_version_non_zero_exit_code(aiida_profile, run_cli_command, monkeypatch, exception_cls, exit_code):
```

Comment:
Was very confused by this. Why are tests in this file prefixed with `tests_` and not `test_` :sweat_smile: 

```suggestion
def test_storage_version_non_zero_exit_code(aiida_profile, run_cli_command, monkeypatch, exception_cls, exit_code):
```

Reply:
Don't know, probably no reason. But there is no real reason to only change this one and don't think changing the entire file for this PR is needed.